uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/BackLayerContent.kt

```diff
@@ -65,12 +65,14 @@ private fun Input(text: String, onClick: () -> Unit) {
                 Spacer(Modifier.width(8.dp))
             }
         }
+
+
```

Comment:
Thank you for contributing!!
Can you delete this blank line?

Reply:
delete blank  line : 062b8eb8356a0aa212e58832e31340f468b08812